app/containers/Vaults/saga.js

```diff
@@ -193,15 +193,21 @@ function* withdrawFromVault(action) {
 }
 
 function* withdrawAllFromVault(action) {
-  const { vaultContract } = action.payload;
+  const { vaultContract, isVaultV2 } = action.payload;
 
   const account = yield select(selectAccount());
 
   try {
     // if (!pureEthereum) {
-    yield call(vaultContract.methods.withdrawAll.cacheSend, {
-      from: account,
-    });
+    if (isVaultV2) {
+      yield call(vaultContract.methods.withdraw.cacheSend, {
+        from: account,
+      });
+    } else {
+      yield call(vaultContract.methods.withdrawAll.cacheSend, {
```

Comment:
Not all v1 vaults have the `withdrawAll` method. I think the correct fix is to never rely on `withdrawAll` and instead do `withdraw(vault.balanceOf(account))`

Reply:
Yeah, the fix was only targeting v2 migratable vaults, and not changing currently working v1 vaults migration implementation, since the reported error was on a wbtc v2 vault.

Reply:
This is a great point, appears this is an issue only for yUSD though, correct? Still, good to update so it works with that one since we'll be migrating it soon.

Reply:
Yes. Better to anticipate on that fact. Pushed the change. So, just lets test that previous migratable vaults continue to work as expected